app/src/mixins/update-user-mixin.ts

```diff
@@ -1,35 +1,48 @@
 import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
 import { UpdateLdUser } from '@/plugins'
 import AuthServices from '@/services/auth-services'
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
 
 @Component({})
 export class UpdateUserMixin extends Vue {
-  /** Fetches the user's info and updates LaunchDarkly. */
-  async updateUser (): Promise<any> {
+  @Getter getKeycloakRoles!: string[]
+
+  /** Fetches the user and org info and updates LaunchDarkly. */
+  async updateLaunchDarkly (): Promise<any> {
     // don't run in Jest tests
     if (process.env.JEST_WORKER_ID) return
 
     try {
-      const userInfo = await AuthServices.fetchUserInfo()
-      if (userInfo) {
-        await this.updateLaunchDarkly(userInfo)
+      // get user info
+      const userInfo = await AuthServices.fetchUserInfo().catch(() => null)
+
+      // get org info
+      const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+      const accountId = currentAccount && JSON.parse(currentAccount)?.id
+      const orgInfo = accountId && await AuthServices.fetchOrgInfo(accountId).catch(() => null)
+
+      const userContext = userInfo && {
+        kind: 'user',
+        // since username is unique, use it as the user key
+        key: userInfo.username,
+        // if we can't get contact email then use user email
+        email: userInfo.contacts[0]?.email || userInfo.email,
+        firstName: userInfo?.firstname,
+        lastName: userInfo?.lastname,
+        roles: this.getKeycloakRoles
+      }
+
+      const orgContext = orgInfo && {
+        kind: 'organization',
+        key: orgInfo.id.toString(),
+        name: orgInfo.name
       }
```

Comment:
Sample org context in LaunchDarkly:

<img width="553" height="406" alt="image" src="https://github.com/user-attachments/assets/99efce29-8c96-459a-a94b-afafe1f122ad" />